triagebot.toml

```diff
@@ -178,6 +178,20 @@ Hi relnotes-interest-group, this PR adds release notes. Could you review this PR
 if you have time? Thanks <3
 """
 
+[ping.codegen-gcc]
+alias = ["cg_gcc"]
+message = """\
+Hey gcc codegen backend team, this issue or PR could use some cg_gcc guidance.
+Could one of you take a look? Thanks <3
+"""
+
+[ping.codegen-cranelift]
+alias = ["cranelift", "cg_cranelift", "cg_clif"]
+message = """\
+Hey cranelift codegen backend team, this issue or PR could use some cg_cranelift
+guidance. Could one of you take a look? Thanks <3
+"""
```

Comment:
Let me know if there's any preferences for aliases, or if I just use change this to accept the canonical `codegen-{cranelift,gcc}` form. I avoided `gcc` alias in general because someday we may wish to setup a custom `gcc` expert ping group.

Reply:
That ship has already sailed with LLVM let's not distract ourselves on hypothetical.

I would also expect an expert ping group to have "expert" in it anyway.

Reply:
It kinda has not. rustbot ping llvm pings the llvm icebreakers ping group, *not* wg-llvm

Reply:
Dropped aliases.